dockerfiles/billing-develop.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:4.5
+
+RUN apt-get update
+RUN apt-get install -y git
+
+RUN mkdir /billing
+WORKDIR /billing
+
+ADD ./dockerfiles/files/billing.config.develop.json /billing/billing.config.develop.json
+RUN mkdir -p $HOME/.storj-billing/config
```

Comment:
`ADD` will create the directory for you: https://docs.docker.com/engine/reference/builder/#/add